app/lib/actions/pulp3/content_view_version/destroy_exporter.rb

```diff
@@ -0,0 +1,16 @@
+module Actions
+  module Pulp3
+    module ContentViewVersion
+      class DestroyExporter < Pulp3::Abstract
+        input_format do
+          param :smart_proxy_id, Integer
+          param :exporter_data, Hash
+        end
+
+        def run
+          ::Katello::Pulp3::ContentViewVersion.new(smart_proxy: smart_proxy).destroy_exporter(input[:exporter_data][:pulp_href])
+        end
```

Comment:
is this an async action?

Reply:
not according to https://docs.pulpproject.org/pulpcore/restapi.html#operation/exporters_core_pulp_exports_delete